src/wallet/rpc/transactions.cpp

```diff
@@ -552,6 +552,8 @@ RPCHelpMan listsinceblock()
                     {"include_removed", RPCArg::Type::BOOL, RPCArg::Default{true}, "Show transactions that were removed due to a reorg in the \"removed\" array\n"
                                                                        "(not guaranteed to work on pruned nodes)"},
                     {"include_change", RPCArg::Type::BOOL, RPCArg::Default{false}, "Also add entries for change outputs.\n"},
+                    {"label", RPCArg::Type::STR, RPCArg::Optional::OMITTED_NAMED_ARG, "If set, should be a valid label name to return only incoming transactions\n"
+                          "with the specified label, or \"*\" to disable filtering and return all transactions."},
```

Comment:
destinations have a label, not transactions.

Reply:
Yes in Bitcoin Core afaik

Reply:
> destinations have a label, not transactions.

What does it mean?

Reply:
Labels are attached to destinations (addresses), not transactions.
`listsinceblock` will list transactions that have destinations with such label.

```suggestion
                    {"label", RPCArg::Type::STR, RPCArg::Optional::OMITTED_NAMED_ARG, "If set, should be a valid label name to return only incoming transactions\n"
                          "with destinations of the specified label, or \"*\" to disable filtering and return all transactions."},
```

Reply:
Yeah, semantic stuff. Thanks, @aureleoules, going to address this improvement in the doc.